### DISCUSSION TITLE
fooocus is not running properly on windows and amd Gpus

### DISCUSSION THREAD

**Discussion by songyifei** [lllyasviel/Fooocus#3288] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/4bb71324-0f39-439c-b1ad-8d136d03f7c4)

**Comment by mashb1t** [comment, marked as answer]
Please ensure to follow https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#windows-amd-gpus step by step and your PC is fulfilling the minimum system requirements.

**Reply by songyifei** [reply]
Thanks for your answer
